packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.module.scss

```diff
@@ -0,0 +1,5 @@
+:local {
+  .visually-hidden-text {
+    height: 0;
```

Comment:
Do you really need this styling?

Reply:
Yes, I think this is needed to not show the vertical scroll bars. cc: @zxeleven

Reply:
See this conversation thread: https://github.com/cerner/terra-framework/pull/1804#discussion_r1372154523